src/renderer/shared/ui-entities/Address/Address.tsx

```diff
@@ -14,32 +14,43 @@ type IconProps = XOR<{
 type Props = IconProps & {
   address: AddressType;
   title?: string;
-  replaceAddressWithTitle?: boolean;
+  hideAddress?: boolean;
   variant?: 'full' | 'truncate' | 'short';
   testId?: string;
 };
 
 export const Address = memo(
-  ({ title, variant = 'full', canCopy = true, showIcon, iconSize = 16, address, testId = 'Address' }: Props) => {
+  ({
+    title,
+    variant = 'full',
+    canCopy = true,
+    showIcon,
+    iconSize = 16,
+    address,
+    testId = 'Address',
+    hideAddress,
```

Comment:
Renamed `replaceAddressWithTitle` to `hideAddress`
If `hideAddress = true` and title is provided as well then hide address
Otherwise `address` becomes `text-text-primary`